src/tests/ftest/server/replay.py

```diff
@@ -0,0 +1,66 @@
+"""
+  (C) Copyright 2019-2023 Intel Corporation.
```

Comment:
Minor - Should this be?
```suggestion
  (C) Copyright 2023 Intel Corporation.
```